cmd/dreamboat/main.go

```diff
@@ -211,12 +211,6 @@ var flags = []cli.Flag{
 		Value:   "",
 		EnvVars: []string{"RELAY_DATAAPI_DATABASE_URL"},
 	},
-	&cli.BoolFlag{ // TODO: Remove
```

Comment:
Don't remove it just yet unless it's already removed from deployment because it would throw an error about unknown parameter